components/utils.py

```diff
@@ -810,6 +822,11 @@ def _get_upgrade_properties(self, service_name):
         with open(upgrade_props_file) as f:
             return json.load(f)
 
+    # XXX reconcile these two with the definitions above
+    def get_install_properties(self, service_name):
```

Comment:
Yes, I know about `.get` - why i wanted to discuss this is, `.get` would give me this file when called with `upgrade_props=True` (because for False, it returns the rollback file, which at this point isn't even created yet).

`.get` would do what i want, but it's somewhat misleading - here i want the original install properties, so passing `upgrade_props=True` seems wrong.

btw, i suspected this might be a bug in `.get`, which is why i prototyped a method here that shows what i need, and left the `# XXX` todo comment :)

So, i should just use `.get` and don't worry about the `upgrade_props` wording?